### PR TITLE
[cpe/alps] mpi.py

### PR DESCRIPTION
**Pull request by jgphpc**
epic https://jira.cscs.ch/browse/VCMSA-263
+ https://jira.cscs.ch/browse/VCMSA-339

### CODE REVIEW

**Comment by jgphpc** [comment]
[ci](https://gitlab.com/cscs-ci/ci-testing/webhook-ci/mirrors/6557018579987861/835515875116200/-/jobs/4343509153#L277): 🆗

File: ci/cpe.yml
Comment (ekouts) [inline ci/cpe.yml:19]: Why are do we skip the autodetection of the remote partitions? Isn't it supposed to be used in our generic tests?

File: checks/prgenv/mpi.py
Comment (ekouts) [inline checks/prgenv/mpi.py:46]: Same for the environment. I would prefer to add a feature for all the mpi envs.

Comment (jgphpc) [reply checks/prgenv/mpi.py]: https://github.com/eth-cscs/cscs-reframe-tests/pull/77/commits/6131d0eb876873aa4bfe70de4cd91597642f3f79

File: checks/prgenv/mpi.py
Comment (jgphpc) [reply checks/prgenv/mpi.py]: https://github.com/eth-cscs/cscs-reframe-tests/pull/77/commits/6131d0eb876873aa4bfe70de4cd91597642f3f79

File: checks/prgenv/mpi.py
Comment (ekouts) [inline checks/prgenv/mpi.py:20]:
```suggestion
    valid_prog_environs = ['+mpi']
    valid_systems = ['+mpi']
```

File: checks/prgenv/mpi.py
Comment (ekouts) [inline checks/prgenv/mpi.py:47]: We can remove this and just add it in the `valid_prog_environs`, `valid_systems`.

File: checks/prgenv/cuda/nvml_check.py
Comment (jgphpc) [inline checks/prgenv/cuda/nvml_check.py:22]: @ekouts : are you ok with that ?

File: config/systems/hohgant.py
Comment (ekouts) [inline config/systems/hohgant.py:185]: This line is not needed right?

**Review by ekouts — CHANGES_REQUESTED** [review]
I have a very small comment, otherwise lgtm and we can merge after

**Review by ekouts — APPROVED** [review]

**Pull request merged**
